spring-xd-reactor/src/main/java/org/springframework/xd/reactor/BroadcasterMessageHandler.java

```diff
@@ -87,21 +90,20 @@
     @SuppressWarnings({"unchecked", "rawtypes"})
     public BroadcasterMessageHandler(Processor processor) {
         Assert.notNull(processor, "processor cannot be null.");
-        this.reactorProcessor = processor;
-        Environment.initializeIfEmpty(); // This by default uses SynchronousDispatcher
 
         this.inputType = ReactorReflectionUtils.extractGeneric(processor);
 
         //Stream with a RingBufferProcessor
-        this.stream = RingBufferProcessor.share("xd-reactor", 8192); //todo expose the backlog size in module conf
+        this.ringBufferProcessor = RingBufferProcessor.share("xd-reactor", ringBufferSize);
```

Comment:
should the name be more unique, maybe add on the classname of the processor impl?